docs/index.md

```diff
@@ -0,0 +1,28 @@
+# Kimai Dockers
+
+We provide a set of docker images for the [Kimai v2](https://github.com/kevinpapst/kimai2) project.
+
+## Quick start
+
+Run the latest dev version of Kimai in production mode using a bundled DB.  **This is not suitable for production use**:
+
+    docker run --rm -ti -p 8001:8001 --name kimai kimai/kimai2:latest-dev
+
+Create an admin user in the new running docker:
+
+    docker exec kimai /opt/kimai/bin/console kimai:create-user admin admin@example.com ROLE_SUPER_ADMIN
+
+This docker transient and will disappear when you stop the container.
+
+## Documentation
+
+ * [Starting a dev instance](dev-instance.md#dev-instances)
```

Comment:
The enumeration violates the [MD007 - Unordered list indentation](https://github.com/DavidAnson/markdownlint/blob/v0.20.4/doc/Rules.md#md007---unordered-list-indentation).

Should be `0` indentations for the main items and `2` indentations for the sub items.